### ISSUE TITLE
Cleanup needed

### ISSUE THREAD

**Issue by TheBlizWiz** [Dylancyclone/VMF2OBJ#10]
Now that the thing is confirmed to work, we ought to clean it up a bit. Ex: We have a gson that we imported but the only time it's ever used is in code that's commented out.

I also suggest we change all the public static final variables at the top to private variables and create some getters/setters for them since having them open like that causes security issues

**Comment by Dylancyclone** [comment]
I agree. Java is very much not my preferred language, and I spend almost all my time working with procedural languages so there's probably a lot of places where better Java/OOP principles can be used. I'll slowly start cleaning stuff up when I notice something is off.
